FWCore/Framework/interface/EventSetup.h

```diff
@@ -33,138 +33,132 @@
 #include "FWCore/Framework/interface/HCMethods.h"
 #include "FWCore/Framework/interface/NoRecordException.h"
 #include "FWCore/Framework/interface/IOVSyncValue.h"
+#include "FWCore/Framework/interface/data_default_record_trait.h"
+#include "FWCore/Utilities/interface/Transition.h"
 
 // forward declarations
 
 namespace edm {
-   class ActivityRegistry;
```

Comment:
Please return this file back to its original spaces to make review easier.

Reply:
Will do.